src/plugins/DeveloperTools.ts

```diff
@@ -193,15 +178,25 @@ if ('__DEV__') {
     }
   };
 } else {
+  // tslint:disable-next-line: max-classes-per-file
+  DeveloperTools = class extends GenericPlugin {
+    public debugOverlay?: HTMLElement;
+
+    public onCreate() {
+      console.warn(`
+        %c [ADJS] ATTENTION! DeveloperTools module has been detected.
+        This plugin is meant to be used in development only. If you wish to enable for your current session,
+        in your console type 'AD_JS.DEBUG()'.
+      `, 'font-weight: bold;');
+    }
+  };
+
   (window as any)[`AD_JS`] = {
-    DEBUG: () => {
-      /*
-        TODO:
-        Since there is no server that we can just fetch
-        the developer bundle from, we need to decide how to get
-        new code to overwrite existing bundle.
-      */
-      return 'This feature is coming soon!. Please use the development build for the time being.';
+    DEBUG: async () => {
+      const url = 'https://unpkg.com/adjs@latest/umd/debug.production.min.js';
+      const options = { async: true, defer: true };
+
+      await loadScript(url, options);
```

Comment:
Maybe we put a console log before this to say “loading.. please wait” so a user knows something is happening and a “finished loading and debug instatiated”